runtime/Cpp/README.md

```diff
@@ -47,6 +47,9 @@ Simply open the VS solution (VS 2013+) and build it.
 #### Compiling on OSX
 Either open the included XCode project and build that or use the cmake compilation as described for linux.
 
+#### Compiling on Android
+Try run cmake -DCMAKE_ANDROID_NDK=/folder/of/android_ndkr17_and_above -DCMAKE_SYSTEM_NAME=Android -DCMAKE_ANDROID_API=14 -DCMAKE_ANDROID_ARCH_ABI=x86 -DCMAKE_ANDROID_STL_TYPE=c++_shared -DCMAKE_ANDROID_NDK_TOOLCHAIN_VERSION=clang -DCMAKE_BUILD_TYPE=Release /folder/antlr4_src_dir -G Ninja.
```

Comment:
Please add a line break after the header.

Reply:
do you think something like html </br> , or just inserting single new line (enter command)?